.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "integration-test/src/test/resources/klaytn-integration-tests"]
 	path = integration-test/src/test/resources/klaytn-integration-tests
 	url = https://github.com/klaytn/klaytn-integration-tests.git
+[submodule "conformance-test/src/test/resources/caver-conformance-tests"]
```

Comment:
This line looks like redundant.
I think this is not a big issue, but just want to share you the issue.

Maybe submodule will work without line 4-6 but need some tests 😅

Reply:
Others all looks great to me :)
Nice and clean structure for conformance tests.

Reply:
Sorry I was mistaken. Not the redundant line, its separate submodule 😓 ...